Sources/AsyncAlgorithms/AsyncChunkedByGroupSequence.swift

```diff
@@ -37,6 +37,7 @@ extension AsyncSequence {
 /// error, `AsyncChunkedByGroupSequence` will rethrow that error immediately and discard
 /// any current group.
 ///
+///      let numbers = [10, 20, 30, 10, 40, 40, 10, 20]
```

Comment:
this should be an async sequence so tack on a `.async` on the end there

Reply:
oops, fixed!